src/build.d

```diff
@@ -163,10 +167,13 @@ Command-line parameters
     if (!flags["DFLAGS"].canFind("-color=off") &&
         [env["HOST_DMD_RUN"], "-color=on", "-h"].tryRun().status == 0)
         flags["DFLAGS"] ~= "-color=on";
-
+    string defaultRule()
+    {
+        return usePGO ? "dmd-pgo" : "dmd";
+    }
```

Comment:
This doesn't need to a function. It's only called in line 176

Reply:
now gone entirely